### PR TITLE
Glue suicide message

### LINKED ISSUE #7798: NO SUICIDE MESSAGE FOR GLUE (GAME BREAKING)
## Reproduction:

GLUE HAS NO SUICIDE MESSAGE. IT IS BORING.
gluicide
IT HAS BEEN TODO FOR ONE YEAR. FIX IT. MAKE IT FUNNY.

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
fixes #7798 
glue has a much better suicide message now

## Why It's Good For The Gamey
closes the issue, its an odd descrepancy to have

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Add a message for glue suicide
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
> sudice

yes

File: code/game/objects/items/devices/glue.dm [hunk 1]
```diff
@@ -6,8 +6,9 @@
 	w_class = WEIGHT_CLASS_SMALL
 	var/uses = 1
 
-/obj/item/syndie_glue/suicide_act(mob/living/carbon/M)
-	return //todo
+/obj/item/syndie_glue/suicide_act(mob/living/M)
+	visible_message("<span class='suicide'>[M] is drinking the whole bottle of glue! It looks like [M.p_theyre()] trying to commit suicide!</span>")
```
Comment (itsmeow) [inline code/game/objects/items/devices/glue.dm]: should be `M.visible_message`

**Review by itsmeow — CHANGES_REQUESTED** [review]
Please fix the awful grammar in the PR body and CL and fill out "Why it's good for the game" properly.

**Comment by PowerfulBacon** [comment]
Sad it doesnt glue their throat shut, disabling their ability to breathe

File: code/game/objects/items/devices/glue.dm (hunk 1 above)
Comment (etherware-novice) [reply code/game/objects/items/devices/glue.dm]: thaats why its not displaying

**Comment by Penwin0** [comment]
Changelog is misspelt

**Comment by etherware-novice** [comment]
alright can someone help? its still displaying the "getting too high on life" message

**Comment by hotimep** [comment]
NEEDS TO BE FUNNIER!!!

**Comment by etherware-novice** [comment]
> NEEDS TO BE FUNNIER!!!

oh shut up you memey dope be glad i dont quit outright

**Review by itsmeow — APPROVED** [review]

**Pull request merged**